resolwe/flow/models.py

```diff
@@ -20,6 +32,43 @@
 .. autoclass:: resolwe.flow.models.Data
     :members:
 
+
+DescriptorSchema model
+**********************
+
+Postgres ORM model for storing descriptors.
+
+.. autoclass:: resolwe.flow.models.DescriptorSchema
+    :members:
+
+
+Process model
+*************
+
+Postgress ORM model for storing processes.
+
+.. autoclass:: resolwe.flow.models.Process
+    :members:
+
+
+Storage model
+*************
+
+Postgres ORM model for storing JSON.
+
+.. autoclass:: resolwe.flow.models.Storage
+    :members:
+
+
+Trigger model
```

Comment:
Remove triggers from docs and code. We do not use them and we do not envision using them in the future. Leftovers from the second generation.